src/components/common/match-item.scss

```diff
@@ -0,0 +1,51 @@
+@import '../../style/common/constants';
+
+.match-item {
+  width: 24vw;
+  display: flex;
+  flex-direction: row;
```

Comment:
```suggestion
```
row it's the default value in web for flex-direction property, you can delete it.